.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 *.mov
+.idea/
```

Comment:
You should not add environment-specific things to the gitignore.
What if I don't use idea?

Please, add such kind of things to your global gitignore file.